torch/cuda/__init__.py

```diff
@@ -362,6 +362,20 @@ def get_device_properties(device: _device_t) -> _CudaDeviceProperties:
         raise AssertionError("Invalid device id")
     return _get_device_properties(device)  # type: ignore[name-defined]
 
+
+def get_alloc_free_events() -> list:
+    r"""Gets allocation/free events sequence for each device.
+
+    Args:
+        None.
+
+    Returns:
+        list: containing alloc/free events for each device alloc/free events
+            of each device is a list of AllocFreeEvent
+    """
+    _lazy_init()  # will define _get_alloc_free_events()
+    return _get_alloc_free_events()  # type: ignore[name-defined]
+
```

Comment:
there are 2 new links between other functions

Reply:
Done. Thank you.